docs/spec/light/api.md

```diff
@@ -0,0 +1,391 @@
+# Cosmos Hub (Gaia) LCD API
+
+This document describes the API that is exposed by the specific LCD implementation of the Cosmos
+Hub (Gaia). Those APIs are exposed by a REST server and can easily be accessed over HTTP/WS(websocket)
+connections.
+
+The complete API is comprised of the sub-APIs of different modules. The modules in the Cosmos Hub
+(Gaia) API are:
+
+* ICS0 (TendermintAPI)
+* ICS1 (KeyAPI)
+* ICS20 (TokenAPI)
+* ICS21 (StakingAPI) - not yet implemented
+* ICS22 (GovernanceAPI) - not yet implemented
+
+Error messages my change and should be only used for display purposes. Error messages should not be
+used for determining the error type.
+
+## ICS0 - TendermintAPI - not yet implemented
```

Comment:
Do we want other endpoints to be part of this ? `/status`, `/net_info`, etc. ?

Reply:
Yes, eventually we'll add those in as well.